docs/Adding_a_Service.rst

```diff
@@ -0,0 +1,100 @@
+| pytest-services/{NEW_SERVICE}                            | create | directory to hold the service specific files               |
++----------------------------------------------------------+--------+------------------------------------------------------------+
+| pytest-services/{NEW_SERVICE}/test_*.py                  | write  | your test cases                                            |
++----------------------------------------------------------+--------+------------------------------------------------------------+
+| pytest-services/{NEW_SERVICE}/resources.py               | write  | Code to fetch & format the data you need to test           |
++----------------------------------------------------------+--------+------------------------------------------------------------+
+| pytest-services/{NEW_SERVICE}/helpers.py                 | write  | Code to simplify writing tests                             |
++----------------------------------------------------------+--------+------------------------------------------------------------+
+| pytest-services/{NEW_SERVICE}/client.py                  | write  | Code to handle connection to the service                   |
++----------------------------------------------------------+--------+------------------------------------------------------------+
+| pytest-services/conftest.py                              | modify | add any new option flags to the pytest invocation          |
++----------------------------------------------------------+--------+------------------------------------------------------------+
+
+In the Mozilla production environment, we invoke the tests inside docker
+containers. To properly configure the docker environment for your service, you'd
+need to also create or modify the following files, which all execute inside the
+base docker container except as noted.
+
++----------------------------------------------------------+--------+------------------------------------------------------------+
+|file                                                      | action | description                                                |
++==========================================================+========+============================================================+
+|foxsec/tools/pytest-services/{NEW_SERVICE}                | create | directory to hold the service specific run files           |
++----------------------------------------------------------+--------+------------------------------------------------------------+
+|foxsec/tools/pytest-services/{NEW_SERVICE}/run.sh         | write  | code to invoke pytest for your service                     |
++----------------------------------------------------------+--------+------------------------------------------------------------+
+|foxsec/tools/pytest-services/{NEW_SERVICE}/config.yaml    | write  | expected failure information                               |
```

Comment:
I think that "expected failure information" as a description is disregarding a lot of options that live in the config file. Might be better to just describe as "pytest-services config file" with a link to the example config, or similar.

Reply:
_Interesting_ -- I thought it was just "exemptions" plus whatever random data my tests needed. 😄 

I see more documentation forthcoming! That's a good thing!